src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixesAndRefactorings/CodeFixContextExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    internal static class CodeFixContextExtensions
+    {
+        /// <summary>
+        /// Remove once https://github.com/dotnet/roslyn/issues/63457 is approved
+        /// </summary>
+        public static bool CanApplyChange(this CodeFixContext context, ApplyChangesKind kind)
+        {
+            var solution = context.Document.Project.Solution;
+#if CODE_STYLE
+            return solution.Workspace.CanApplyChange(kind);
```

Comment:
still accessing teh workspace here.  but ideally only temporarily.  once https://github.com/dotnet/roslyn/issues/63457 is approved we'll have a public API that our own code, and the CODE_STYLE layer can use.